bot/seasons/evergreen/bookmark.py

```diff
@@ -0,0 +1,76 @@
+import logging
+
+import discord
+from discord.ext import commands
+
+log = logging.getLogger(__name__)
+
+
+class Bookmark(commands.Cog):
+    """A cog for Bookmarking a message."""
+
+    def __init__(self, bot: commands.Bot):
+        self.bot = bot
+
+    @commands.command(name="bookmark", aliases=("bm", "pin"))
+    async def bookmark(self, ctx: commands.Context, jump_url: str = None, *args) -> None:
+        """Bookmarks a message."""
+        if jump_url is not None and jump_url != "*":
+            if "discordapp.com/channels" not in jump_url:
+                await ctx.send("I can't find the associated message,"
+                               " the command support the following syntax:\n"
+                               " `.bm [message url] (bookmark name)`")

```

Comment:
See my other comment on multiline string joins for what's wrong with this.

Additional things that are wrong with it:
1. `the command support` -> `this command supports`
2. Comma should probably be replaced by a period.

You also shouldn't hardcode the prefix.